pkg/trace/log/buflogger.go

```diff
@@ -93,3 +95,8 @@ func (b *buflogger) Criticalf(format string, params ...interface{}) error {
 
 // Flush implements Logger.
 func (b *buflogger) Flush() {}
+
+// GetLogLevel implements Logger.
+func (b *buflogger) GetLogLevel() (log.Level, error) {
+	return log.DebugLvl, nil
```

Comment:
Why debug? Shouldn't it return the actual log level?

Reply:
Looks like the `buflogger` is only used for testing.

Reply:
The buflogger doesn't have a log level, it just always logs everything.

Reply:
Got it, let's add a code comment then explaining why it always returns Debug.